src/components/AddActivity.tsx

```diff
@@ -26,85 +28,90 @@ export default function AddActivity() {
-                      required
-                      id="outlined-required"
-                      label="Sted"
-                      />
+                        <TextField
+                          required
+                          id="outlined-required"
+                          label="Sted"
+                        />
 
-                      <TextField
-                      required
-                      id="outlined-required"
-                      label="Beskrivelse av aktivitet"
-                      multiline
-                      rows={9}
-                      />
-                  </div>
+                        <TextField
+                          required
+                          id="outlined-required"
+                          label="Beskrivelse av aktivitet"
+                          multiline
+                          rows={9}
+                        />
+                    </div>
+                    
                   </Box>
               </FormControl>
 
-              <Grid container spacing={2} sx={{ ml: '5px'}}>
-                <Grid item xs={1}>
+              <Grid container spacing={2} sx={{ ml: '1px', mt: '5px', mb: '8px'}}>
+                <Grid item xs={3}>
                   <h4>Last opp et bilde:</h4>
                 </Grid>
-
                 <Grid item xs={2}>
                   <input type="file"
-                  id="fileUploadButton"
-                  style={{ display: 'none' }}
-                  // onChange={onFileChange}
+                    id="fileUploadButton"
+                    style={{ display: 'none' }}
+                    // onChange={onFileChange}
                   />
-
                   <label htmlFor={'fileUploadButton'}>
                     <Button
-                        variant="contained"
-                        component="span"
-                        sx={{ backgroundColor: 'darkgrey' }}
+                      variant="contained"
+                      component="span"
+                      sx={{ backgroundColor: 'darkgrey' }}
                     >
+                      <ImageIcon sx={{ mr: '10px', verticalAlign: 'middle' }} />
                       Velg fil
                     </Button>
                   </label>
                 </Grid>
               </Grid>
 
-              <Grid container spacing={2} sx={{ ml: '5px'}}>
-                <Grid item xs={1}>
+              <Grid container spacing={2} sx={{ ml: '1px'}}>
+                <Grid item xs={3}>
                   <h4>Velg et bilde:</h4>
                 </Grid>
-
                 <Grid item xs={2}>
-                    Bilde alternativer
+                    <p>Bilde alternativer</p>
                 </Grid>
               </Grid>
 
-              <Box  sx={{ display: 'flex', justifyContent: 'flex-end'}}>
-                <Button variant='contained' sx={{ backgroundColor: 'darkgrey' }}>Legg til aktivitet</Button>
+              <Box sx={{ display: 'flex', justifyContent: 'flex-end'}}>
```

Comment:
As I mentioned: it was quite a while since I use react mui. This is the only way to style these mui components?
(preferably these should be separated in their own css / scss etc. file)